src/acceptanceTest/java/uk/gov/hmcts/futurehearings/pip/acceptance/publications/GETPublicationsTest.java

```diff
@@ -1,19 +1,13 @@
 package uk.gov.hmcts.futurehearings.pip.acceptance.publications;
```

Comment:
Could you please explain the reason(s) to have this test file with no test in it?

Reply:
These were the template files, just removed the tests can remove the file

Reply:
> These were the template files, just removed the tests can remove the file

Did you mean that you cannot remove the file as it to be used as template?